spring-cloud-config-client/pom.xml

```diff
@@ -21,6 +21,11 @@
     ]]>
 	</description>
 
+	<properties>
+		<jasypt-spring-boot.version>3.0.5</jasypt-spring-boot.version>
+		<okhttp.version>4.11.0</okhttp.version>
```

Comment:
Can we use the boot-managed version?

Reply:
@spencergibb 
I attempted "boot managed" JCE but had problems with cipher in the client properties. There is so little information on JCE in a Spring Boot project and tons of examples for JASYPT that I went with the common encryption that was easier to set up and manage. 

I can revisit it if I can get better documentation of getting it working the the config client.

```
client-secret: '{cipher}e4b1f56fb017319eb959595343d0e4f5a742e975860b723433f90e8f3e869a379db7c756e0537273aebf72cc3a4c3d48caf4955889c5e22be588523b0a88b5c8'

    Property: spring.cloud.config.client-secret
    Value: "{cipher}e4b1f56fb017319eb959595343d0e4f5a742e975860b723433f90e8f3e869a379db7c756e0537273aebf72cc3a4c3d48caf4955889c5e22be588523b0a88b5c8"
    Origin: URL [file:config/application-local.yml] - 73:22
    Reason: java.lang.UnsupportedOperationException: No decryption for FailsafeTextEncryptor. Did you configure the keystore correctly?
```

Reply:
Mock server for IDP testing. See ConfigClientRequestTemplateFactoryTest

Reply:
Ok, then back to my original comment, why can't the boot managed version of ok http be used?

Reply:
Sorry, looking back I didn't specify okhttp as the one to boot manage.

Reply:
Nope https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/3.2.0/spring-boot-dependencies-3.2.0.pom

```
<okhttp.version>4.12.0</okhttp.version>
```

You should just be able to remove the version property and the `version` tag.

Reply:
Done. Did not know it was included. Will have to update all my microservices.

Reply:
It didn't used to be, it was added at some point. We had the same thing in spring cloud